### PR TITLE
Initialize Rails app

### PR DESCRIPTION
**Pull request by ismail-syed**
Pretty self-explanatory here. Initialized a standard rails app with a `README.md`. Please go over the readme to verify you can get setup. 

If you think we can shred off some of the extra boilerplate files, let me know 

Fixes https://github.com/GraphQL-Query-Planner/project/issues/1

### CODE REVIEW

**Comment by ismail-syed** [comment]
@DerekStride @justinkrol. 
Github wouldn't let me set you two as reviewers. Don't know if it's permission related to hiccups on Github's side.

**Comment by ismail-syed** [comment]
@DerekStride @justinkrol. 
Github wouldn't let me set you two as reviewers. Don't know if it's permission related to hiccups on Github's side.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+* chruby: v0.3.9
+* ruby-install: v0.6.1
+* Ruby: v2.4.1
+* RubyGems: v2.6.12
+* Rails: v5.1
+* MySQL: v5.7.19
+
+
+
+## Getting Started
```
Comment (justinkrol) [inline README.md:15]: I don't think we need this section, but I don't mind keeping it. Maybe also add this?
> Install dependencies.
`bundle install`

**Review by justinkrol — APPROVED** [review]

**Comment by justinkrol** [comment]
BTW I don't think this fully closes #1. Will also need a PR/commit to add `graphql-ruby`

**Comment by justinkrol** [comment]
Closing. Can you reopen this in the new webscale repo?

**Comment by ismail-syed** [comment]
Yup, thanks!

**Pull request closed without merging**